client/influxdb.go

```diff
@@ -145,6 +145,27 @@ func (c *Client) Ping() (time.Duration, string, error) {
 	return time.Since(now), version, nil
 }
 
+func (c *Client) Dump(db string) (*http.Response, error) {
```

Comment:
We also might want to add this after prompt for password so if they want to do this interactively they could.  Not 100% sure it matters that much, but it's an easy enough use case to support.